guides/common/modules/proc_importing-all-available-ansible-playbooks.adoc

```diff
@@ -2,17 +2,24 @@
 = Importing all available Ansible playbooks
 
 You can import all the available Ansible playbooks to {Project} from collections installed on {SmartProxy}.
+{Project} creates job templates from the imported playbooks and places the templates in the `Ansible Playbook - Imported` job category.
+
+If you have a custom collection, place it in `/etc/ansible/collections/ansible_collections/_My_Namespace_/_My_Collection_`.
 
 .Prerequisites
-* Ansible plugin in {Project} is enabled
+* Ansible plugin in {Project} is enabled.
+* Your {Project} account has a role that grants the `import_ansible_playbooks` permission.
 
 .Procedure
 // Not available via Hammer/UI: https://projects.theforeman.org/issues/34318
 * Import the Ansible playbooks using the following API request:
 +
 [options="nowrap", subs="+quotes,verbatim,attributes"]
 ----
-# curl -X PUT 'Content-Type: application/json' https://_{foreman-example-com}_/ansible/api/v2/ansible_playbooks/sync?proxy_id=_My-{smart-proxy-context}-ID_
+# curl -X PUT -H 'Content-Type: application/json' https://_{foreman-example-com}_/ansible/api/v2/ansible_playbooks/sync?proxy_id=__My_{smart-proxy-context}_ID__
```

Comment:
When the -H argument wasn't explicitly provided, curl was reporting a malformed-url warning.